extensions/amp-story/1.0/amp-story.css

```diff
@@ -183,13 +183,24 @@ amp-story amp-sidebar[side][open] {
   max-width: 280px !important;
   min-width: 280px !important;
   width: 280px !important;
+  transform: translate3d(0, 0, 0) !important;
+  transition: transform 0.2s cubic-bezier(0.4, 0.0, 1, 1) !important;
```

Comment:
Oops, I think that animation is meant to hide the sidebar? It's set on the `[open]` attribute, could you please swap these open/close animations?

Reply:
Thank you! Done